viewer/packages/cad-parsers/src/cad/filterPrimitivesV9.test.ts

```diff
@@ -0,0 +1,365 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+
+import { filterGeometryOutsideClipBox } from './filterPrimitivesV9';
+import { RevealGeometryCollectionType } from '@reveal/sector-parser';
+import * as THREE from 'three';
+
+import {
+  PrimitiveType,
+  Box,
+  Circle,
+  Cone,
+  EccentricCone,
+  Ellipsoid,
+  GeneralCylinder,
+  GeneralRing,
+  Quad,
+  Torus,
+  Trapezium,
+  Nut,
+  getCollectionType
+} from '../../../../test-utilities/src/primitives/primitiveTypes';
+
+import {
+  createPrimitiveInterleavedGeometry,
+  parseInterleavedGeometry
+} from '../../../../test-utilities/src/primitives/primitiveThreeBuffers';
+
+function testSecondFilteredAwayFromBoxAtX10Yneg10Z0(primitives: any[], primitiveType: PrimitiveType) {
```

Comment:
Can `primitives` be changed to something more exact?

Thinking of something in the lines of
```

type Primitive = Box | Circle | Nut | ...;
function testSecondFilteredAwayFromBoxAtX10Yneg10Z0(primitives: Primitive[], primitiveType: PrimitiveType) {
```

I know it really doesn't matter, but it makes the code 2% more readble